components/org.wso2.carbon.identity.api.server.input.validation/org.wso2.carbon.identity.api.server.input.validation.v1/src/main/resources/inputValidation.yaml

```diff
@@ -15,10 +15,12 @@ info:
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
 
 servers:
-  - url: 'https://{host-name}/api/server/v1'
+  - url: 'https://{server-url}/t/{tenant-domain}/api/server/v1'
```

Comment:
Dont we need to generate the files for this change?

Reply:
I don't think we need, the change doesn't contain any change on schema. Where do u think, this change will reflect?

Reply:
Ack. Since this is just the server URL change, wont be an issue